packages/modules/src/scoring/Scoring.ts

```diff
@@ -0,0 +1,55 @@
+import { SCORE_MODULUS, IJournalContent } from '@cord.network/types'
+import { Crypto } from '@cord.network/utils'
+import {
+  SCORE_IDENT,
+  SCORE_PREFIX,
+  ScoreType,
+  IScoreAverageDetails,
+} from '@cord.network/types'
+import { Identifier } from '@cord.network/utils'
+import { fetchScore } from './Scoring.chain'
+
+export function adjustAndRoundRating(rating: number): number {
+  rating = Math.round(rating * SCORE_MODULUS)
+  return rating
+}
+
+export function ComputeActualRating(rating: number): number {
+  return rating / SCORE_MODULUS
+}
+
+export function ComputeAverageRating(rating: number, count: number): number {
+  return rating / count
+}
+
+export function generateDigestFromJournalContent(
+  journalContent: IJournalContent
+) {
+  const digest = Crypto.hash(JSON.stringify(journalContent))
+  const hexDigest = Crypto.u8aToHex(digest)
+  return hexDigest
```

Comment:
We need a function, that will take a score input and produce the transformed content that can be used with the toChain function to anchor the details to the chain. The return value should have all the elements required for the extrinsic

Reply:
Have made a separate PR for this issue. Please review: https://github.com/dhiway/cord.js/pull/135